app/views/shared/_project_summary.html.erb

```diff
@@ -21,6 +21,10 @@
             row.key { t("project.summary.region.title") }
             row.value { @project.establishment.region_name }
           end
+          summary_list.row do |row|
+            row.key { t("project.summary.sharepoint_link.title") }
+            row.value { link_to(@project.sharepoint_link, @project.sharepoint_link, target: :_blank) }
```

Comment:
Same comments any time we use this, maybe the presenter could make this simpler?